### ISSUE TITLE
Pseudo Label 

### ISSUE THREAD

**Issue by hulianyuyy** [ycmin95/VAC_CSLR#6]
I'm wondering how to assign labels for frames with CTC loss. It seems CTC Loss can be viewed as sequential SoftMax losses. But the key point is how to obtain the pseudo labels for frames via back propagation. Thanks in advance!

**Comment by ycmin95** [comment]
You can refer to [this repo](https://github.com/vadimkantorov/ctc/blob/master/ctc.py#L88), which calculate the gradients of CTC via PyTorch autograd instead of the forward-backward algorithm.

**Comment by hulianyuyy** [comment]
Thanks again for your kind reply.
